app/error.jsx

```
@@ -0,0 +1,33 @@
+      <h1 className="text-7xl font-black font-sans">Oops</h1>
+      <h2 className="text-3xl md:text-5xl font-bold font-sans">
+        Something went wrong,
+      </h2>
+      <h3 className="text-3xl md:text-5xl font-bold font-sans">
+        and that's on us.
+      </h3>
+      <h4 className="text-rose-500">{error.message}</h4>
+      <button
+        onClick={reset}
+        className="text-blue-800 bg-blue-100 shadow-sm rounded-lg px-5 py-2 font-bold"
+      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)